experimental/framework/react-inputs/tsconfig.json

```diff
@@ -1,10 +1,9 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../common/build/build-common/tsconfig.node16.json",
 	"exclude": ["dist", "node_modules"],
```

Comment:
should we add lib to this?

Reply:
I don't think so. I do think these `exclude`s are all useless since `include` only picks up `./src/...`. Not having `lib` in the changes and not having problems is further evidence that they are not useful in standard configurations. I just wasn't going to add one more change to the general changes and hadn't suggested removal. Good eye. :)